h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -299,6 +296,12 @@ public static BoosterParms createParams(XGBoostParameters p, int nClasses) {
     }
     params.put("nthread", nthread);
 
+    Log.info("XGBoost Parameters:");
+    for (Map.Entry<String,Object> s : params.entrySet()) {
+      Log.info(" " + s.getKey() + " = " + s.getValue());
+    }
+    Log.info("");
```

Comment:
Why ?

Reply:
Yes, that was on purpose - it actually looks nice in the log ;)